### PR TITLE
React: Create top-up services in place using config fetched from Relayer

### PR DESCRIPTION
**Pull request by msieczko**

### CODE REVIEW

File: universal-login-react/src/ui/TopUp/ChooseTopUpMethod.tsx [hunk 1]
```diff
@@ -36,7 +38,7 @@ export const ChooseTopUpMethod = ({contractAddress, onPayClick, topUpClassName,
                 onClick={() => setTopUpMethod('crypto')}
                 checked={topUpMethod === 'crypto'}
                 name="top-up-method"
-                className={`top-up-method ${topUpMethod === 'crypto' ? 'active' : '' }`}
+                className={`top-up-method ${topUpMethod === 'crypto' ? 'active' : ''}`}
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/TopUp/ChooseTopUpMethod.tsx]: extract `activeClassFor(topUpMethod === 'crypto', 'active')`
@alexcherman 
In one of following PRs

File: universal-login-react/src/ui/TopUp/ChooseTopUpMethod.tsx [hunk 2]
```diff
@@ -50,7 +52,7 @@ export const ChooseTopUpMethod = ({contractAddress, onPayClick, topUpClassName,
                 onClick={() => setTopUpMethod('fiat')}
                 checked={topUpMethod === 'fiat'}
                 name="top-up-method"
-                className={`top-up-method ${topUpMethod === 'fiat' ? 'active' : '' }`}
+                className={`top-up-method ${topUpMethod === 'fiat' ? 'active' : ''}`}
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/TopUp/ChooseTopUpMethod.tsx]: extract `activeClassFor(topUpMethod === 'fiat', 'active')`

File: universal-login-react/src/ui/TopUp/ChooseTopUpMethod.tsx (hunk 1 above)
Comment (alexcherman) [reply universal-login-react/src/ui/TopUp/ChooseTopUpMethod.tsx]: ok

File: universal-login-react/src/ui/TopUp/TopUp.tsx [hunk 3]
```diff
@@ -40,6 +47,7 @@ export const TopUp = ({sdk, onGasParametersChanged, contractAddress, startModal,
           <ChooseTopUpMethod
             contractAddress={contractAddress}
             onPayClick={onPayClick}
+            ipGeolocationApiConfig={ipGeolocationApiConfig}
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/TopUp/TopUp.tsx]: Why not skip all the passing of values and just do:
```ts
ipGeolocationApiConfig={sdk.realyerConfig.ipGeolocationApiConfig}
```

Same for `onRampConfig`?

Comment (msieczko) [reply universal-login-react/src/ui/TopUp/TopUp.tsx]: done in https://github.com/UniversalLogin/UniversalLoginSDK/pull/1201

**Pull request merged**